docs/api/sql/Function.md

```diff
@@ -381,6 +381,32 @@ SELECT ST_Centroid(polygondf.countyshape)
 FROM polygondf
 ```
 
+## ST_ClosestPoint
+
+Introduction: Returns the 2-dimensional point on geom1 that is closest to geom2. This is the first point of the shortest line between the geometries. If using 3D geometries, the Z coordinates will be ignored. If you have a 3D Geometry, you may prefer to use ST_3DClosestPoint.
+
```

Comment:
Update documentation here to include the thrown exception if any of the geometry is empty

Reply:
I tried to resolve all your comments! You can review again.  :) Thanks!